__tests__/update-state.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { Runner } from "..";
+
+describe(`Running on ${Runner.getNetworkFromEnv()}`, () => {
+  let runner: Runner;
+  jest.setTimeout(60000);
+
+  beforeAll(async () => {
+    runner = await Runner.create(async ({ runtime }) => {
+      const contract = await runtime.createAndDeploy(
+        "status-message",
+        `${__dirname}/build/debug/status_message.wasm`
+      );
+      const ali = await runtime.createAccount("ali");
+      return { contract, ali };
+    });
```

Comment:
May as well make the implicit return our official practice throughout this codebase

```suggestion
    runner = await Runner.create(async ({ runtime }) => ({
      contract: await runtime.createAndDeploy(
        "status-message",
        `${__dirname}/build/debug/status_message.wasm`
      ),
      ali: await runtime.createAccount("ali")
    }));
```

Reply:
Yeah but we need to implement deploy and init so that this can stay simple in other tests.